todo_app/templates/index.html

```diff
@@ -3,16 +3,100 @@
 
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-4 text-center">DevOps Engineering Tasks</h1>
+    <!--<p class="lead text-center">DevOps Engineering Tasks</p> -->
+    <br>
+    <br>
+
+   <h5 class="text-center" >Create A New Task</h5> 
+    <form method="post" action="/add_tasks">
+      <div class="form-group">
+     <label for="title">Task:</label>
+        <input type="text" class="form-control" id="title" placeholder="Type A New Task" name="title">
+      </div>
+
+      <div class="form-group">
+        <label for="priority">Select Priority</label>
+        <select class="form-control" id="priority" name="priority">
+          <option>P1</option>
+          <option>P2</option>
+          <option>P3</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+        <label for="status">Select Status</label>
+        <select class="form-control" id="status" name="status">
+          <option>In Progress</option>
+          <option>Not Started</option>
+          <option>Completed</option>
+          <option>Deferred</option>
+        </select>
+      </div>
+
+      <button type="submit" class="btn btn-primary"> Create A New Task</button>
+    </form>
+    <br>
+    <br>
+
+
+  <h5 class="text-center">Update Task Status</h5> 
+    <form method="post" action="update_task_status">
+      <div class="form-group ">
+     <label for="id">Task ID:</label>
+        <input type="number" class="form-control" id="id" placeholder="Enter Task ID #" name="id">
+      </div>
+
+    <div class="form-group">
+      <label for="status">Select Status</label>
+      <select class="form-control" id="status" name="status">
+        <option>In Progress</option>
+        <option>Not Started</option>
+        <option>Completed</option>
+        <option>Deferred</option>
+      </select>
+    </div>
+    <button type="submit" class="btn btn-primary">Update Task Status</button>
+    </form>
+
+    <br>
+    <br>
+
+     <h5 class="text-center">Delete A Task</h5> 
+    <form method="post" action="delete_task">
+      <div class="form-group">
+        <label for="id">Task ID:</label>
+        <input type="number" class="form-control" id="id" placeholder="Enter Task ID #" name="id">
```

Comment:
Adding a `required` keyword to some of these inputs would be nice, to get some basic browser support to stop people accidentally submitting no id, and then seeing an error:
```suggestion
        <input type="number" class="form-control" id="id" placeholder="Enter Task ID #" name="id" required>
```